src/solve.jl

```diff
@@ -343,15 +343,15 @@ function solve!(integrator::DDEIntegrator)
             typeof(prob),typeof(integrator.sol.alg),typeof(interp)}(
                 sol_array.u, u_analytic, errors, sol_array.t, interp.ks,
                 prob, integrator.sol.alg, interp, interp.alg_choice,
-                interp.dense, integrator.sol.tslocation, :Success)
+                interp.dense, integrator.sol.tslocation, integrator.integrator.sol.destats, :Success)
```

Comment:
why from integrator.integrator?

Reply:
`integrator.sol.destats` gives 0 for some reason.

Reply:
isn't `integrator` being stepped, and `integrator.integrator` just being a store? @devmotion

Reply:
Yes, `integrator` is stepped but the solution in `integrator.integrator` is used for inter- and extrapolation and updated in https://github.com/JuliaDiffEq/DelayDiffEq.jl/blob/master/src/integrator_interface.jl#L25. Shouldn't you get `integrator.sol.destats == integrator.integrator.sol.destats` if you specify `destats` explicitly in https://github.com/JuliaDiffEq/DelayDiffEq.jl/blob/master/src/solve.jl#L72-L80?

Reply:
Yes, that would do it. I agree that the correct thing to do would be to alias the destats here